content/05-mysql/04-connecting-to-mysql-databases.mdx

```diff
@@ -116,3 +116,55 @@ If you want to modify the rules that dictate how users can authenticate to your
 In this guide, we covered MySQL authentication from the client side.  We demonstrated how to use the `mysql` command line client to connect to both local and remote database instances.
 
 Knowing how to connect to various MySQL instances is one of the first steps you need to understand as you start to work the database system.  You may run a local MySQL instance for development that doesn't need any special authentication, but your databases in staging and production will almost certainly require authentication.  Being able to authenticate in either case will allow you to work well in different environments.
+
+## FAQ
+
+<details><summary>Can you connect to a MySQL database in Python?</summary>
+
+Yes, you can connect to a MySQL database using Python. You will need to download the MySQL Python connector and use the `connect()` constructor.
+
+MySQL provides a [developer guide](https://dev.mysql.com/doc/connector-python/en/connector-python-example-connecting.html) once the connector is downloaded.
+
+</details>
+
+<details><summary>How do you allow MySQL to accept remote connections?</summary>
```

Comment:
This seems like a different question than the one being answered.  At least the way I interpreted it, this is asking what you must change on the server to enable external connections.